src/Microsoft.SqlTools.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -343,6 +343,14 @@ internal static CompletionItem[] EmptyCompletionList
         }
 
         /// <summary>
+        /// Retrieves the reserved word list 
+        /// </summary>
+        public static string[] GetReservedWordList()
+        {
+            return DefaultCompletionText;
```

Comment:
>DefaultCompletionText [](start = 19, length = 21)

I think we can just make DefaultCompletionText internal instead of adding this method.